src/vivarium_gates_mncnh/data/hemoglobin_effects/hgb_lbwsg_effect_generation.py

```diff
@@ -0,0 +1,371 @@
+import os
+
+import numpy as np
+import pandas as pd
+import scipy
+from vivarium import Artifact, InteractiveContext
+
+
+def load_bop_rrs(outcome):
+    """Load burden of proof hemoglobin estimates for specified outcome"""
+    # TODO: move these files into model repo?
+    rrs = pd.read_csv(f"/mnt/team/anemia/pub/bop/sim_studies/{outcome}/inner_draws.csv")
```

Comment:
add these to `src/model_repo/data/hemglobin_effects/` but make the file names more informative when we add them